### PR TITLE
feat: new rules and updates

### PR DESCRIPTION
**Pull request by nasbench**
### Summary of the Pull Request

This PR introduces new rules and some update to older rules

### Detailed Description of the Pull Request / Additional Comments

- Added new rules related to RogueRDP
- Added new rules and updates related to Winget
- Updated browser list for some rules
- General quality of life updates to some older rules.

### Example Log Event

N/A

### Fixed Issues

N/A

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

File: rules/windows/network_connection/net_connection_win_binary_susp_com.yml [hunk 1]
```diff
@@ -1,49 +1,60 @@
...
+            - 'paste.ee'
+            - 'pastebin.pl'
+            - 'pastetext.net'
             - 'privatlab.com'
+            - 'privatlab.net'
+            - 'send.exploit.in'
             - 'sendspace.com'
-            - 'pastetext.net'
-            - 'pastebin.pl'
-            - 'paste.ee'
```
Comment (phantinuss) [inline rules/windows/network_connection/net_connection_win_binary_susp_com.yml:42]: would anything ending on `/` work? I don't have my machine up and running to verify. Like is it really only the host name or the URL?

File: rules/windows/process_creation/proc_creation_win_winget_local_install_via_manifest.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+title: Install New Package Via Winget Local Manifest
+id: 313d6012-51a0-4d93-8dfc-de8553239e25
+status: experimental
+description: |
+    Detects usage of winget to install applications via manifest file. Adversaries can abuse winget to download payloads remotely and execute them.
+    The manifest option enables you to install an application by passing in a YAML file directly to the client.
+    Winget can be used to download and install exe's, msi, msix files later.
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_winget_local_install_via_manifest.yml:7]:
```suggestion
    Winget can be used to download and install exe, msi or msix files later.
```

File: rules/windows/registry/registry_set/registry_set_winget_enable_local_manifest.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+title: Enable Local Manifest Installation With Winget
+id: fa277e82-9b78-42dd-b05c-05555c7b6015
+status: experimental
+description: Detects changes to the AppInstaller (winget) policy specifically the activation of the local manifest installation. Which allows a user to install new packages via custom manifests.
```
Comment (phantinuss) [inline rules/windows/registry/registry_set/registry_set_winget_enable_local_manifest.yml:4]:
```suggestion
description: Detects changes to the AppInstaller (winget) policy. Specifically the activation of the local manifest installation, which allows a user to install new packages via custom manifests.
```

File: rules/windows/network_connection/net_connection_win_binary_susp_com.yml (hunk 1 above)
Comment (phantinuss) [reply rules/windows/network_connection/net_connection_win_binary_susp_com.yml:42]: I suspect we have to strip the non-domain parts, don't we?

File: rules/windows/process_creation/proc_creation_win_mstsc_run_local_rdp_file_susp_location.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        # Note: This list of paths is better transformed into a whitelist where you only exclude legitimate locations you use in your env
+        CommandLine|contains:
+            - ':\Users\Public\'
+            - ':\Windows\System32\spool\drivers\color'
+            - ':\Windows\System32\Tasks_Migrated '
+            - ':\Windows\Tasks\'
+            - ':\Windows\Temp\'
+            - ':\Windows\Tracing\'
+            - '\AppData\Local\Temp\'
+            - '\Desktop\' # Could be source of FP depending on the environement
```
[the remark this answers is not in the document]
Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_mstsc_run_local_rdp_file_susp_location.yml:34]: I put the FP comment and said I would try my luck 😂

File: rules/windows/network_connection/net_connection_win_binary_susp_com.yml (hunk 1 above)
Comment (nasbench) [reply rules/windows/network_connection/net_connection_win_binary_susp_com.yml:42]: Good catch. It indeed shows domains/hostnames not URLs. I'll make the necessary adjustments.

**Review by phantinuss — APPROVED** [review]

**Review by Neo23x0 — APPROVED** [review]

**Pull request merged**